### ISSUE TITLE
OSX conda compilation/import error

### ISSUE THREAD

**Issue by elyase** [salestock/fastText.py#42]
In conda python 3.5, OSX El Capitan 10.11.14 I am getting: 

```
(fasttext)❯ python -c "import fasttext"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/Users/elyase/miniconda3/envs/fasttext/lib/python3.5/site-packages/fasttext/__init__.py", line 1, in <module>
    from .fasttext import skipgram
ImportError: dlopen(/Users/elyase/miniconda3/envs/fasttext/lib/python3.5/site-packages/fasttext/fasttext.cpython-35m-darwin.so, 2): Symbol not found: __ZNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEED1Ev
  Referenced from: /Users/elyase/miniconda3/envs/fasttext/lib/python3.5/site-packages/fasttext/fasttext.cpython-35m-darwin.so
  Expected in: flat namespace
 in /Users/elyase/miniconda3/envs/fasttext/lib/python3.5/site-packages/fasttext/fasttext.cpython-35m-darwin.so
```

```
(fasttext)❯ python --version
Python 3.5.2 :: Continuum Analytics, Inc.
```

**Comment by elyase** [comment]
Solved it by adding `-mmacosx-version-min=10.9'` to `setup.py`, the final line looks like this:

```
        extra_compile_args=['-pthread', '-funroll-loops', '-std=c++0x', '-mmacosx-version-min=10.9'])
```

**Comment by pyk** [comment]
Hi @elyase , thanks for the report!
